winlogbeat/sys/wineventlog/renderer.go

```diff
@@ -258,7 +251,10 @@ func (r *Renderer) renderUser(handle EvtHandle, event *winevent.Event) (values [
 	// Fingerprint the argument types to help ensure we match these values with
 	// the correct event data parameter names.
 	argumentHash := xxhash.New()
-	binary.Write(argumentHash, binary.LittleEndian, propertyCount)
+	err = binary.Write(argumentHash, binary.LittleEndian, int64(propertyCount))
+	if err != nil {
+		return nil, 0, fmt.Errorf("failed to hash property count: %w", err)
+	}
```

Comment:
This would never have contributed to the hash, though given that the event var types would have, this will likely not have caused failures.

Reply:
For reference, this would have been returning `binary.Write failed: binary.Write: invalid type int`.